src/parser/rogue/outlaw/modules/azerite/Deadshot.js

```diff
@@ -0,0 +1,60 @@
+//This module looks the Outlaw Azerite trait 'Deadshot' which gives a flat damage buff to pistol shot after 'Between the Eyes' has been used.  
+//This trait has a 100% chance on cast to provide the damage buff.
+
+import React from 'react';
+import SPELLS from 'common/SPELLS'; 
+import Analyzer from 'parser/core/Analyzer';
+import { formatNumber } from 'common/format';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+import { calculateAzeriteEffects } from 'common/stats';
+import calculateBonusAzeriteDamage from 'parser/core/calculateBonusAzeriteDamage';
+import ItemDamageDone from 'interface/others/ItemDamageDone';
+import StatTracker from 'parser/shared/modules/StatTracker';
+
+const ES_AP_COEFFICIENT = 0.35135; //Coefficient is taken from Pistolshot AP coefficient 
```

Comment:
Make the variable/constant name something more fitting, maybe 
```suggestion
const PISTOL_SHOT_COEFFICIENT = 0.35135; // Coefficient is taken from Pistol Shot AP coefficient 
```

Reply:
Corrected.  Now reads:

```const PISTOL_SHOT_COEFFICIENT........```  

Also changed the related coefficient on line 43 to work with this rename.